internal/objects/service/service.go

```diff
@@ -94,14 +92,14 @@ var (
 	//  https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
 	//
 	InternalLBAnnotations = map[operatorv1alpha1.LoadBalancerProviderType]map[string]string{
-		awsProviderType: {
+		operatorv1alpha1.AWSLoadBalancerProvider: {
 			awsInternalLBAnnotation: "0.0.0.0/0",
```

Comment:
I believe this should be changed from `0.0.0.0/0` to `"true"`. Ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer